README.md

```diff
@@ -44,27 +44,34 @@ npm config set externalplugins ../relative/path/to/plugins/directory
 
 ## Adding npm Package Dependencies
 
-If you want to add a new npm package to 'node_modules':
+If you want to add a new npm package to 'node_modules' you will need to `--save-exact`:
 
 1. Install the new package
-  * Install and save dependencies in `package.json`
+  * The command below will install and save dependencies in `package.json`
 
     ```
-    npm install [your package] --save
+    npm install [your package] --save --save-exact
     ```
 
-  * Then, add the package to devDependencies
+  * Then, (if needed) add the package to devDependencies
 
     ```
-    npm install [your package] --save-dev
+    npm install [your package] --save-dev --save-exact
     ```
 
 2. Create a synced npm-shrinkwrap.json with devDependencies included by running
 
     ```
+    npm shrinkwrap
+    ```
+and/or
+
+    ```
     npm shrinkwrap --dev
     ```
 
+ps: after generating the `shrinkwrap` you will need to remove the `fsevents` from the updates since it's automatically added and not needed.
```

Comment:
We have a `fixShrinkwrap` script wich runs when you run `npm run shrinkwrap`, which takes care of the `fsevents` part. You only need to manually remove it if shrinkwrap runs automatically.